azkaban-common/src/main/java/azkaban/executor/ExecutionFlowDao.java

```diff
@@ -84,7 +87,8 @@ public void uploadExecutableFlow(final ExecutableFlow flow)
     final SQLTransaction<Long> insertAndGetLastID = transOperator -> {
       transOperator.update(INSERT_EXECUTABLE_FLOW, flow.getProjectId(),
           flow.getFlowId(), flow.getVersion(), flow.getStatus().getNumVal(),
-          submitTime, flow.getSubmitUser(), submitTime, executorId, flowPriority, executionSource);
+          submitTime, flow.getSubmitUser(), submitTime, executorId, flowPriority, executionSource
+          , dispatchMethod.getNumVal());
```

Comment:
I assume you are keeping num value for the optimization that int will provide from database side instead of enum name itself.